src/std/text/hex.ss

```diff
@@ -23,39 +23,46 @@
 (def (hex u4)
   (string-ref hexes u4))
 
-(def unhexes
-  (let (ht (make-hash-table-eq))
-    (for-each (cut hash-put! ht <> <>)
-              (string->list "0123456789")
-              (iota 10))
-    (for-each (cut hash-put! ht <> <>)
-              (string->list "abcdef")
-              (iota 6 10))
-    (for-each (cut hash-put! ht <> <>)
-              (string->list "ABCDEF")
-              (iota 6 10))
-    ht))
+(def 0-code (char->integer #\0))
+(def unhexes-len (- (char->integer #\g) 0-code))
+(def unhexes (make-u8vector unhexes-len 255))
+(def (unhex-index c) (##fx- (char->integer c) 0-code))
+(let loop ((i 0))
+  (when (<= i 15)
+    (let ((c (hex i)))
+      (##u8vector-set! unhexes (unhex-index c) i)
+      (##u8vector-set! unhexes (unhex-index (char-upcase c)) i))
+    (loop (1+ i))))
+
+(def (unhex* char)
+  (def i (unhex-index c))
```

Comment:
did you mean `(unhex-index char)`? travis build fails with unbound identifier here.